index.html

```diff
@@ -178,8 +180,12 @@ <h3>Recommendation Track specification repositories</h3>
       <h3>Test repositories</h3>
       <ul>
         <li><a href="https://github.com/w3c/imsc-tests/pulse">IMSC Tests</a></li>
+        <li><a href="https://github.com/w3c/IMSC-1.0.1_Text_TestContent/pulse">Test contents for IMSC 1.0.1 Text</a></li>
+        <li><a href="https://github.com/w3c/IMSC-1.1_Text_TestContent/pulse">Test contents for IMSC 1.1 Text</a></li>
+        <li><a href="https://github.com/w3c/IMSC-1.1_Image_TestContent/pulse">Test contents for IMSC 1.1 Image</a></li>
         <li><a href="https://github.com/w3c/ttml1-tests/pulse">TTML1 tests</a></li>
         <li><a href="https://github.com/w3c/ttml2-tests/pulse">TTML2 tests</a></li>
+        <li><a href="https://github.com/w3c/ttml3-tests/pulse">TTML3 tests</a></li>
```

Comment:
I think a goal for TPAC would be to decide what to do with TTML3. So far, it seems that the 2nd edition of TTML2, the extension modules and the new version of IMSC would be sufficient.

Reply:
removed per TPAC discussion
> in terms of goals, we decided not to progress TTML3 for the time being